service/src/main/java/org/cbioportal/service/impl/StructuralVariantServiceImpl.java

```diff
@@ -36,12 +39,32 @@
 
     @Autowired
     private StructuralVariantRepository structuralVariantRepository;
-    
+
+    @Autowired
+    private AlterationEnrichmentUtil alterationEnrichmentUtil;
+
     @Override
     public List<StructuralVariant> fetchStructuralVariants(List<String> molecularProfileIds, 
             List<Integer> entrezGeneIds,  List<String> sampleIds) {
         
         return structuralVariantRepository.fetchStructuralVariants(molecularProfileIds, entrezGeneIds, sampleIds);
     }
 
+    @Override
+    public List<StructuralVariantCountByGene> getSampleCountInMultipleMolecularProfiles(List<String> molecularProfileIds,
+                                                                               List<String> sampleIds, List<Integer> entrezGeneIds, boolean includeFrequency) {
```

Comment:
Lets adjust the whitespace here and break the argument list into some additional lines.

Reply:
adjusted.